src/Ocelot/Configuration/Creator/UpstreamTemplatePatternCreator.cs

```diff
@@ -5,7 +5,7 @@ namespace Ocelot.Configuration.Creator
 {
     public class UpstreamTemplatePatternCreator : IUpstreamTemplatePatternCreator
     {
-        private const string RegExMatchOneOrMoreOfEverything = ".+";
+        private const string RegExMatchZeroOrMoreOfEverything = ".*";

```

Comment:
Seems we need to make this constant as `public` because we need it for tests.
Otherwise you have to care about them manually.
We have to add /embed this regex value as a part of testing templates to show a reader (code reviewer) that those test are dependent much from this RegEx. It will be easy to read and support the tests in future.

Reply:
@raman-m you are correct I will make it public because all the test cases use this